CONTRIBUTING.md

```diff
@@ -81,6 +81,15 @@ Run `./runServer.sh` in `ui`
 - Clone [nussknacker-quickstart](https://github.com/TouK/nussknacker-quickstart)
 - Run `docker-compose -f docker-compose-env.yml -f docker-compose-custom.yml up -d` inside it
 
+#### Setting up Kubernetes environment
+
+To run streaming lite scenarios with K8s, we recommend using [k3d](https://k3d.io) with
+[nussknacker-quickstart](https://github.com/TouK/nussknacker-quickstart) setup
+- Run integration environment, as described above
+- `export K3D_FIX_DNS=1; k3d cluster create --network nussknacker_network` - this will create K8s cluster, which
+  has access to docker network used by integration environment. [K3D_FIX_DNS](https://github.com/rancher/k3d/issues/209)
+- run `buildAndImportRuntimeImageToK3d` in sbt (can be skipped if you intend to use e.g. `latest-staging` docker image) 
```

Comment:
1. Can it be just:
> run `sbt buildAndImportRuntimeImageToK3d` 
, instead of run ... in sbt?
2. staging-latest

Reply:
1. it looks like it not working

Reply:
:+1: